keyboards/choc_taro/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "choc_taro",
+    "url": "https://kakunpc.booth.pm/",
+    "maintainer": "kakunpc",
+    "width": 15,
+    "height": 5,
+    "layouts": {
+        "LAYOUT_all": {
+            "layout": [{"label":"1", "x":0, "y":0}, {"label":"2", "x":1, "y":0}, {"label":"3", "x":2, "y":0}, {"label":"4", "x":3, "y":0}, {"label":"5", "x":4, "y":0}, {"label":"6", "x":5, "y":0}, {"label":"7", "x":6, "y":0}, {"label":"8", "x":7, "y":0}, {"label":"9", "x":8, "y":0}, {"label":"10", "x":9, "y":0}, {"label":"11", "x":10, "y":0}, {"label":"12", "x":11, "y":0}, {"label":"13", "x":12, "y":0}, {"label":"14", "x":13, "y":0}, {"label":"15", "x":14, "y":0}, {"label":"16", "x":0, "y":1, "w":1.5}, {"label":"17", "x":1.5, "y":1}, {"label":"18", "x":2.5, "y":1}, {"label":"19", "x":3.5, "y":1}, {"label":"20", "x":4.5, "y":1}, {"label":"21", "x":5.5, "y":1}, {"label":"22", "x":6.5, "y":1}, {"label":"23", "x":7.5, "y":1}, {"label":"24", "x":8.5, "y":1}, {"label":"25", "x":9.5, "y":1}, {"label":"26", "x":10.5, "y":1}, {"label":"27", "x":11.5, "y":1}, {"label":"28", "x":12.5, "y":1}, {"label":"29_1", "x":13.5, "y":1, "w":1.5}, {"label":"30", "x":0, "y":2, "w":1.75}, {"label":"31", "x":1.75, "y":2}, {"label":"32", "x":2.75, "y":2}, {"label":"33", "x":3.75, "y":2}, {"label":"34", "x":4.75, "y":2}, {"label":"35", "x":5.75, "y":2}, {"label":"36", "x":6.75, "y":2}, {"label":"37", "x":7.75, "y":2}, {"label":"38", "x":8.75, "y":2}, {"label":"39", "x":9.75, "y":2}, {"label":"40", "x":10.75, "y":2}, {"label":"41", "x":11.75, "y":2}, {"label":"42", "x":12.75, "y":2, "w":2.25}, {"label":"43", "x":0, "y":3, "w":2.25}, {"label":"44", "x":2.25, "y":3}, {"label":"45", "x":3.25, "y":3}, {"label":"46", "x":4.25, "y":3}, {"label":"47", "x":5.25, "y":3}, {"label":"48", "x":6.25, "y":3}, {"label":"49", "x":7.25, "y":3}, {"label":"50", "x":8.25, "y":3}, {"label":"51", "x":9.25, "y":3}, {"label":"52", "x":10.25, "y":3}, {"label":"53", "x":11.25, "y":3}, {"label":"54", "x":12.25, "y":3, "w":1.75}, {"label":"55", "x":14, "y":3}, {"label":"56", "x":1.25, "y":4, "w":1.25}, {"label":"57", "x":2.5, "y":4, "w":1.25}, {"label":"58", "x":3.75, "y":4, "w":2.25}, {"label":"59", "x":6, "y":4, "w":1.25}, {"label":"60", "x":7.25, "y":4, "w":2.75}, {"label":"61", "x":10, "y":4, "w":1.25}, {"label":"62", "x":11.25, "y":4, "w":1.25}, {"label":"63", "x":12.5, "y":4, "w":1.25}]
```

Comment:
This is more like nitpicking rather than an issue, but could you reorder the `layout` in a similar way like this: https://github.com/qmk/qmk_firmware/blob/master/keyboards/dz60/info.json

1 key per line and an empty line between rows.

Reply:
Can you do this so fauxpark won't have to do it later? :D

Reply:
Okey.

Reply:
Do I need to add the layout of ISOEnter with reference to DZ60?

Reply:
Just add the layouts this board supports. Erovia is more referring to the style here, than copying layouts form the DZ60's info.json.

Reply:
Yeah, sorry I wasn't clear. I just wanted to give you an example on the preferred styling.

Reply:
Thank you.
Added supporting layouts.